### ISSUE TITLE
Misleading semantic HTML in relevance bar

### ISSUE THREAD

**Issue by lukavdplas** [UUDigitalHumanitieslab/I-analyzer#1260]
Tiny nitpick: the relevance indicator is a `<progress>` element in the HTML but it should be `<meter>`.
